_posts/minutes/2016-11-23-23.md

```diff
@@ -0,0 +1,44 @@
+---
+    title: 'Minutes of Meeting on 23/11/2016'
+    author: qaisjp
+    category: minutes
+---
+This week's minutes were taken by Joshua Green.
```

Comment:
Yeah, just scratch it. It doesn't really matter who wrote the minutes as long as they were taken